Framework/Algorithms/test/MaxEntTest.h

```diff
@@ -279,6 +279,30 @@ class MaxEntTest : public CxxTest::TestSuite {
     TS_ASSERT_THROWS_ANYTHING(alg->execute());
   }
 
+  void test_adjustments_summed_too_few_spectra() {
+
+    auto ws = WorkspaceCreationHelper::create2DWorkspace(6, 10);
+    auto ws1 = WorkspaceCreationHelper::create2DWorkspace(2, 10);
+
+    IAlgorithm_sptr alg = AlgorithmManager::Instance().create("MaxEnt");
+    alg->initialize();
+    alg->setChild(true);
+    alg->setProperty("InputWorkspace", ws);
+    alg->setProperty("ComplexData", true);
+    alg->setPropertyValue("MaxIterations", "1");
+    // We need as many spectra in the adjustments as
+    // in the input workspace even though images are summed.
+    alg->setProperty("DataLinearAdj", ws1);
+    alg->setProperty("DataConstAdj", ws1);
+    alg->setProperty("PerSpectrumReconstruction", false);
+    alg->setPropertyValue("ReconstructedImage", "image");
+    alg->setPropertyValue("ReconstructedData", "data");
+    alg->setPropertyValue("EvolChi", "evolChi");
+    alg->setPropertyValue("EvolAngle", "evolAngle");
+
+    TS_ASSERT_THROWS_ANYTHING(alg->execute());
```

Comment:
we should be catching the type of exception thrown

Reply:
Can you give a useful example for `alg->execute()`?